### ISSUE TITLE
Note id comparison broken

### ISSUE THREAD

**Issue by ibrokemypie** [ibrokemypie/mastalab#7]
Mostly disabled for now
expects id to be a long integer, but it is a string
should replace with createdAt comparison

need to at least disable the rest as they cause crashes

**Comment by ghost** [comment]
You should use dates.

**Comment by ibrokemypie** [comment]
i totally agree, just dont really know how yet/havent looked into it as it seems rather big

**Comment by ghost** [comment]
I would suggest to add a parameter in Status Class like:
`long time_id ` and in the Api Class, we can transform the date in a timestamp (millisecond) to save it in `time_id`. Then these `time_id `will replace the current toot ids (for comparaisons only, because it doesn't have sense for Misskey itself).
